### PR TITLE
Include example template + check hook output

### PR DESCRIPTION
**Pull request by asachs01**
This solves for #27 and #30 by providing a workaround in the form of a template + check hook output that can be put into the template body until the plugin is refactored so that hook output and templates aren't mutually exclusive.

### CODE REVIEW

**Comment by nixwiz** [comment]
Still needs some format changes.  How about this?

```<html>
Greetings,<br>
<br>
<h3>Informational Details</h3>
<b>Check</b>: {{ .Check.Name }}<br>
<b>Entity</b>: {{ .Entity.Name }}<br>
<b>State</b>: {{ .Check.State }}<br>
<b>Occurrences</b>: {{ .Check.Occurrences }}<br>
<b>Playbook</b>: https://example.com/monitoring/wiki/playbook<br>
<h3>Check Output Details</h3>
<b>Check Output</b>: {{.Check.Output}}<br>
<h4>Check Hook(s)</h4>
{{range .Check.Hooks}}<b>Hook Name</b>:  {{.Name}}<br>
<b>Hook Command</b>:  {{.Command}}<br>
<b>Hook Output</b>: {{.Output}}<br>
{{end}}

<br>
#monitoringlove,<br>
<br>
Sensu<br>
</html>
```

However, this does bring to light one issue.  When using HTML, the formatting of the hook output (or any multi-line string) won't be properly formatted.  Here's an example from above.

Screen Shot 2019-09-05 at 2 56 42 PM

**Comment by asachs01** [comment]
@cwjohnston can I get a 👍 on this?

File: README.md [hunk 1]
```diff
@@ -1,44 +1,21 @@
+[![Bonsai Asset Badge](https://img.shields.io/badge/Sensu%20Go%20Email%20Handler-Download%20Me-brightgreen.svg?colorB=89C967&logo=sensu)](https://bonsai.sensu.io/assets/sensu/sensu-email-handler) TravisCI: [![TravisCI Build Status](https://travis-ci.org/sensu/sensu-email-handler.svg?branch=master)](https://travis-ci.org/sensu/sensu-email-handler)
+
 # Sensu Go Email Handler Plugin
-TravisCI: [![TravisCI Build Status](https://travis-ci.org/sensu/sensu-email-handler.svg?branch=master)](https://travis-ci.org/sensu/sensu-email-handler)
+- [Overview](#overview)
+- [Usage examples](#usage-examples)
+- [Configuration](#configuration)
+  - [Asset registration](#asset-registration)
+  - [Asset manifest](#asset-manifest)
```
Comment (cwjohnston) [inline README.md]:
```suggestion
  - [Asset definition](#asset-definition)
```

File: README.md [hunk 2]
```diff
@@ -1,44 +1,21 @@
...
+  - [Handler manifest](#handler-manifest)
```
Comment (cwjohnston) [inline README.md]:
```suggestion
  - [Handler definition](#handler-definition)
```

File: README.md [hunk 3]
```diff
@@ -63,6 +40,61 @@ Flags:
...
+
+### Asset registration
+
+Assets are the best way to make use of this handler. If you're not using an asset, please consider doing so! If you're using sensuctl 5.13 or later, you can use the following command to add the asset: 
+
+`sensuctl asset add sensu/sensu-email-handler`
+
+If you're using an earlier version of sensuctl, you can download the asset definition from [this project's Bonsai Asset Index page](https://bonsai.sensu.io/assets/sensu/sensu-email-handler).
+
+### Asset manifest
```
Comment (cwjohnston) [inline README.md]:
```suggestion
### Asset definition
```

File: README.md [hunk 4]
```diff
@@ -63,6 +40,61 @@ Flags:
...
+    io.sensu.bonsai.tags: handler
+spec:
+  url: https://assets.bonsai.sensu.io/45eaac0851501a19475a94016a4f8f9688a280f6/sensu-email-handler_0.2.0_linux_amd64.tar.gz
+  sha512: d69df76612b74acd64aef8eed2ae10d985f6073f9b014c8115b7896ed86786128c20249fd370f30672bf9a11b041a99adb05e3a23342d3ad80d0c346ec23a946
+  filters:
+  - entity.system.os == 'linux'
+  - entity.system.arch == 'amd64'
+```
+
+### Handler manifest
```
Comment (cwjohnston) [inline README.md]:
```suggestion
### Handler definition
```

**Comment by asachs01** [comment]
@cwjohnston can you take a gander again plz?

**Pull request merged**